Setup.md

```diff
@@ -33,24 +36,34 @@ You will be redirected to the application page where you can access your Client
 
 To add the Client ID and Client Secret to the App, follow these steps:
 
-1. In Xcode, go to `Product` (in the Menu bar) > `Scheme` > `Manage Schemes...`
-2. Select `Freetime-AppCenter` and click the cog.
-3. Click on `Duplicate`.
-4. Uncheck the `Shared` checkbox.
-5. Select `Copy of Freetime-AppCenter` and click on `Edit...`
-6. Go to `Run` > `Arguments`
-7. Add your Client ID (`GITHUB_CLIENT_ID` as key) and Client Secret (`GITHUB_CLIENT_SECRET`) to the Environment Variables.
+1. In the directory you created, open the {YOUR_DIR}/GitHawk/Freetime.xcworkspace file.
+2. In Xcode, go to `Product` (in the Menu bar) > `Scheme` > `Manage Schemes...`
+3. Select `Freetime-AppCenter` and click the cog.
+4. Click on `Duplicate`.
+5. Select `Copy of Freetime-AppCenter` and uncheck "Shared" checkbox. 
+6. With 'Copy of Freetime-Appcenter' selected, click on `Edit...`
+7. Go to `Run` > `Arguments`
+8. Add your Client ID (`GITHUB_CLIENT_ID` as key) and Client Secret (`GITHUB_CLIENT_SECRET`) to the Environment Variables.
 
 The copy of the scheme is to avoid commiting your private environement variable to the repository.
 
+>   Checkpoint: At this point you should be able to successfully build the app on the simulator
+
+🎉 The project is officially set up! 🎉 
+
+At this point you can begin making changes and testing on your simulator.
+If you would like to test on a physical device, one more step remains.
+
 ## 4. Setting up code-signing (With automatic code-signing)
 
+> ⚠️ **1. This step is only if you would like to test/run the app on your physical device 2. This step changes the bundleID and groupID, keep in mind when you commit to leave these changes out**
```

Comment:
Did you intend for this line to contain both numbers in the list?

Reply:
It’s a block quote. So it doesn’t work as a list I think.
I think it’s okay

Reply:
It looks great!

Reply:
Yeah that doesn't seem to work indeed. Should be OK though :)